pyproject.toml

```diff
@@ -105,8 +103,8 @@ test = [
     "pytest-cov",
     "pytest-profiling",
     "pytest-xdist",
-    'importlib-metadata; python_version <"3.8.0"',
-    'nbmake <1.3; python_version=="3.7.*"',
+    'importlib-metadata; python_version <"3.9.0"',
+    'nbmake <1.3; python_version=="3.9.*"',
```

Comment:
Same, these requirements are redundant now. BUT, make sure that `nbmake` is still listed as a requirement.